frontend/src/components/Create.js

```diff
@@ -128,6 +128,9 @@ class Create extends Component {
 
     handleSave = () => {
         alert("Attempting to save");
+        var dayDate = new Date().getDate(); //Current Date
```

Comment:
Use `let` instead of `const`. 
Also since you are creating dates 3 times, it might be weird. Maybe better to do something like
something like
let date = new Date()
let day = date.getDate()
....